src/Extensions.cs

```diff
@@ -370,9 +380,19 @@ public static bool HasInterface(this IModelType type)
         /// <returns></returns>
         public static string GetInterfaceName(this IModelType type, bool includePkgName = false)
         {
+            // this function is called in a *LOT* of places so is perf sensitive

```

Comment:
when you say this function you mean  `GetInterfaceName` i presume

Reply:
Correct.